src/Terminal/getTabCompletionPossibilities.ts

```diff
@@ -324,6 +325,7 @@ export async function getTabCompletionPossibilities(terminalText: string, baseDi
         .map((server) => server.hostname),
       scripts: [...currServ.scripts.keys()],
       txts: [...currServ.textFiles.keys()],
+      enums: structuredClone(enums),
```

Comment:
Is it possible for enums to be a frozen object such that we don't need to clone it? This isn't in the critical path for module creation, it only gets called for autocomplete, but it would be nice to keep the number of objects created down, if possible.

If there's a technical reason why that isn't easy, this is fine.

Reply:
`enums` is frozen in `src\NetscriptFunctions.ts`, so it's fine to use `enums: enums`. I used it like that at first, but before committing, I got paranoid about `Object.freeze`, so I added `structuredClone`. The usage of `Object.freeze` in `NetscriptFunctions.ts` looks fine, though.